articles/scopes/preview/index.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Scopes
+description: Overview of scopes.
+toc: true
+---
+
+# Scopes
+
+<div class="alert alert-info">
+<strong>Heads up!</strong> This article describes the latest Auth0 implementation, according to the <a href="/api-auth">API Authorization flows</a>. If you are looking for the legacy Scopes doc refer to <a href="/scopes">Scopes</a>.
+</div>
+
+In OpenID Connect (OIDC) we have the notion of __scopes__, or __claims__. These are strings, sent as part of the `scope` request parameter. These claims can be standard (as defined by the specification) or custom.
+
+
+Scopes are used in the following cases:
+
+- When you want to get an `id_token` in the response. In this case, you need to include `openid` as part of the `scope` request parameter: `scope=openid`.
+
+- When you want to get additional user information, like email or picture. For details refer to [Standard Claims](#standard-claims).
+
+- When you want to access an API. In this case, you need to define custom scopes for your API and add these values  as part of the `scope` request parameter, for example: `scope=read:contacts`. For details refer to [API Scopes](#api-scopes).
+
+
+## Standard Claims
+
+OpenID Connect specifies a set of [standard claims](https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims). These are user attributes, intended to supply the client with user details such as email, name and picture.
+
+This information can be returned in the `id_token` or in the response from [the /userinfo endpoint](/api/authentication#get-user-info).
+
+__NOTE__: The `updated_at` claim is returned as a [Unix timestamp](https://en.wikipedia.org/wiki/Unix_time) instead of an [ISO 8601 date string](https://en.wikipedia.org/wiki/ISO_8601) for consistency with the `exp`, `iat` and `nbf` claims.
```

Comment:
We need to double check with @jhiner about this. I'm not sure if its an epoch or not.